Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        private readonly string _name;
+        private readonly decimal _amount;
+        
+
+        public Person(string name, decimal amount)
+        {
+            _name = name;
+            _amount = amount;
+        }
+
+        public decimal GetAmount()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetAmount' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)